test/run-drun/timer.mo

```diff
@@ -0,0 +1,42 @@
+//MOC-ENV MOC_UNLOCK_PRIM=yesplease
+
+import { debugPrint; error; setTimer; cancelTimer } = "mo:⛔";
+
+actor {
+
+  system func timer() : async () {
+      await @run_timers();
+  };
+
+  var count = 0;
+  var max = 10;
+  let raw_rand = (actor "aaaaa-aa" : actor { raw_rand : () -> async Blob }).raw_rand;
+
+  public shared func go() : async () {
+     var attempts = 0;
+
+     let id1 = setTimer(1, false, func () : async () { count += 1; debugPrint "YEP!" });
```

Comment:
FileCheck that the `global_timer` calls are there?

Conversely have a `no-timer.mo` test that is `-no-timer` flagged, and FileChecks that there is no such thing.